jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -468,6 +468,18 @@
       <repository location="http://download.jboss.org/jbosstools/updates/requirements/egit/v4.0.0.201505260635-rc2/"/>
       <unit id="org.eclipse.egit.ui.importer" version="4.0.0.201505271301"/>
     </location>
+    
+    <!-- Eclipse Docker Tooling -->
+    <location includeAllPlatforms="true" includeMode="slicer" type="InstallableUnit" includeSource="true">
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/docker/"/>
```

Comment:
We need the version suffix in mirror URL here: http://download.jboss.org/jbosstools/updates/requirements/docker/1.0.0.201506030402/